app/src/main/java/app/revanced/integrations/youtube/settings/Settings.java

```diff
@@ -135,6 +135,7 @@ public class Settings extends BaseSettings {
     public static final BooleanSetting HIDE_GRAY_SEPARATOR = new BooleanSetting("revanced_hide_gray_separator", TRUE);
     public static final BooleanSetting HIDE_SNACK_BAR = new BooleanSetting("revanced_hide_snack_bar", FALSE);
     public static final BooleanSetting REMOVE_VIEWER_DISCRETION_DIALOG = new BooleanSetting("revanced_remove_viewer_discretion_dialog", FALSE);
+    public static final BooleanSetting HOOK_PLAYLIST_DOWNLOAD_BUTTON = new BooleanSetting("revanced_hook_playlist_download_button", TRUE);
```

Comment:
The name of the setting that hooks the video download button is `Settings.EXTERNAL_DOWNLOADER_ACTION_BUTTON`, but the name of the setting that hooks the playlist download button is `Settings.HOOK_PLAYLIST_DOWNLOAD_BUTTON`.

It is better to change the name of the settings to a more similar structure.